crates/node-core/src/node_config.rs

```diff
@@ -239,21 +239,15 @@ impl NodeConfig {
     }
 
     /// Returns the initial pipeline target, based on whether or not the node is running in
-    /// `debug.tip` mode, `debug.continuous` mode, or neither.
+    /// `debug.tip` mode or neither.
     ///
     /// If running in `debug.tip` mode, the configured tip is returned.
-    /// Otherwise, if running in `debug.continuous` mode, the genesis hash is returned.
     /// Otherwise, `None` is returned. This is what the node will do by default.
-    pub fn initial_pipeline_target(&self, genesis_hash: B256) -> Option<B256> {
+    pub fn initial_pipeline_target(&self) -> Option<B256> {
         if let Some(tip) = self.debug.tip {
```

Comment:
this can just be `self.debug.tip`, which means we can remove this function and just use `self.debug.tip` where this is invoked instead